meta-resin-common/recipes-core/images/resin-image-initramfs.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Small image capable of booting a device. The kernel includes \
+the Minimal RAM-based Initial Root Filesystem (initramfs), which finds the \
+first 'init' program more efficiently. This is used to prepare the rootfs \
+with some key operations."
+
+PACKAGE_INSTALL = "busybox udev base-passwd ${ROOTFS_BOOTSTRAP_INSTALL}"
+
+# Do not pollute the initrd image with rootfs features
+IMAGE_FEATURES = ""
+
+export IMAGE_BASENAME = "resin-image-initramfs"
+IMAGE_LINGUAS = ""
+
+LICENSE = "MIT"
+
+IMAGE_FSTYPES = "${INITRAMFS_FSTYPES}"
+inherit core-image
+
+IMAGE_ROOTFS_SIZE = "8192"
```

Comment:
@floion it was ack - he added it but I reverted it because I needed more space for integrating the expander in initramfs. So I was forced to switch back to 8M.